public/js/Pull.js

```diff
@@ -24,15 +24,12 @@ _.extend(Pull.prototype, {
          groups.user = null;
 
          signatures.forEach(function(signature) {
-            if (signature.data.active) {
-               groups.current.push(signature);
-               users[signature.data.user.id] = true;
-
-               if (utils.mySig(signature)) {
-                  groups.user = signature;
+            if (!users[signature.data.user.id]) {
```

Comment:
I would invert this and early return to reduce nesting.

Reply:
Good idea!
3a87c82